regex-assembly/932380.ra

```diff
@@ -0,0 +1,26 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
+##! Word list for rule 932380 (RCE Windows command injection part 2/2)
+##!
+##!+ i
+
+##! Note: the quoting prefixes are part of the command prefixes, except for ^
```

Comment:
This command belongs in `windows-commands-prefix.ra`.

Reply:
Done